src/thinblock.cpp

```diff
@@ -1777,6 +1777,13 @@ void BuildSeededBloomFilter(CBloomFilter &filterMemPool,
             }
         }
     }
+    else // Add all the transaction hashes currently in the mempool
+    {
+        std::vector<uint256> vMempoolHashes;
+        mempool.queryHashes(vMempoolHashes);
+        setHighScoreMemPoolHashes.insert(vMempoolHashes.begin(), vMempoolHashes.end());
+    }
+
```

Comment:
won't change anything, but wouldn't it be logic if this commit got merged w/ the previous one? I mean, in case this PR will be merged and not squashed by @gandrewstone, leaving the structure of the commits as is would break `git bisect` in case we need to use it to hunt a bug.

Reply:
sure i can do that